Makefile

```diff
@@ -72,7 +72,7 @@ update_code:
 	git checkout master && git pull
 
 update_deps:
-	pip install --no-deps --exists-action=w -r requirements/dev.txt --find-links https://pyrepo.addons.mozilla.org/wheelhouse/ --find-links https://pyrepo.addons.mozilla.org/ --no-index
+	pip install --no-deps --exists-action=w -r requirements/dev.txt --find-links https://pyrepo.addons.mozilla.org/wheelhouse/ --find-links https://pyrepo.addons.mozilla.org/
```

Comment:
DO NOT MERGE (it's just to make sure the tests run correctly on travis without the pytest and pytest-django versions on pyrepo